pom.xml

```diff
@@ -52,16 +52,14 @@
   <dependencies>
 
     <dependency>
-      <groupId>org.cyberneko</groupId>
-      <artifactId>html</artifactId>
+      <groupId>org.codelibs</groupId>
+      <artifactId>nekohtml</artifactId>
+      <version>2.1.1</version>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.core.runtime</groupId>
-      <artifactId>eclipse-core-runtime</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>xerces</groupId>
-      <artifactId>xercesImpl</artifactId>
```

Comment:
Removing the xerces dependency also makes the corresponding `dependencyManagement` entry obsolete.